osmaxx/excerptexport/models/extraction_order.py

```diff
@@ -146,6 +147,38 @@ def get_absolute_url(self):
         from django.core.urlresolvers import reverse
         return reverse('excerptexport:status', kwargs={'extraction_order_id': self.id})
 
+    def send_email_if_all_exports_done(self, incoming_request):
+        if all(export.is_status_final for export in self.exports.all()):
```

Comment:
correct english is quite high expectations for a programming language :laughing: